resources/styles/book-content/ap-connection.less

```diff
@@ -0,0 +1,9 @@
+.ap-connection {
+  > h1:first-child {
+    // The "Connection for AP Courses" message appears on every page and is redundant.
+    // although we don't want it to display, it still should take up space in order to provide a top margin
+    height: 0;
+    margin: 30px;
+    visibility: hidden;
```

Comment:
@Fredasaurus Is this what was decided in the discussion? My understanding was that it was suggested but then kept in